
From: <blaisorblade_spam@yahoo.it>

Avoid redeclaring again (resulting in a compilation error) cpu_online and
cpu_*_map, which are now declared elsewhere.

Signed-off-by: Paolo 'Blaisorblade' Giarrusso <blaisorblade_spam@yahoo.it>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/asm-um/smp.h |    6 ------
 1 files changed, 6 deletions(-)

diff -puN include/asm-um/smp.h~uml-dont-declare-cpu_online-fix-compilation-error include/asm-um/smp.h
--- 25/include/asm-um/smp.h~uml-dont-declare-cpu_online-fix-compilation-error	Fri Oct  8 14:50:50 2004
+++ 25-akpm/include/asm-um/smp.h	Fri Oct  8 14:50:50 2004
@@ -8,10 +8,6 @@
 #include "asm/current.h"
 #include "linux/cpumask.h"
 
-extern cpumask_t cpu_online_map;
-extern cpumask_t cpu_possible_map;
-
-
 #define smp_processor_id() (current_thread->cpu)
 #define cpu_logical_map(n) (n)
 #define cpu_number_map(n) (n)
@@ -19,8 +15,6 @@ extern cpumask_t cpu_possible_map;
 extern int hard_smp_processor_id(void);
 #define NO_PROC_ID -1
 
-#define cpu_online(cpu) cpu_isset(cpu, cpu_online_map)
-
 extern int ncpus;
 
 
_
